README.md

```diff
@@ -1 +1,42 @@
-# geckos-14
\ No newline at end of file
+# DashThings.io
+
+DashThing.io is a tool dedicated to DIYers in need of an easy way to build
```

Comment:
(suggestion)
DashThing.io is a web app that allows visualisation of data read from different devices. It is a dashboard that gives DIYers a straightforward way to observe and analyse experiment process and results.